tasks/55_find-object-analogy/find-object-analogy.js

```diff
@@ -0,0 +1,9 @@
+export const whatIsInAName = (arr, obj) => {
```

Comment:
FYI  - I think here you could use memoization technique to avoid O(n^2) complexity.